TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Components/Intake.java

```diff
@@ -0,0 +1,44 @@
+package org.firstinspires.ftc.teamcode.Components;
+
+import com.qualcomm.robotcore.eventloop.opmode.OpMode;
+import com.qualcomm.robotcore.hardware.Servo;
+
+/**
+ * Intake Controls:
+ * Gamepad: 2
+ * Intake Button: A
+ * Outtake Button: B
+ */
+public class Intake {
+
+    OpMode opMode;
+    Servo motor;
+
+    private final double speed = 1;
+
+    public Intake(OpMode opMode){
+        this.opMode = opMode;
+    }
+
+    public void init(boolean debug){
+        motor = opMode.hardwareMap.servo.get("Intake");
+
+        if (debug){
+            opMode.telemetry.addData("INFO", "Intake INIT");
+        }
+    }
+
```

Comment:
Just a note, we do have to configure the servos tomorrow. This is hardware though, not software

Reply:
@GarrettBurroughs I have no idea so... probably yes